sys/net/gnrc/pktdump/gnrc_pktdump.c

```diff
@@ -101,10 +107,17 @@ static void _dump_snip(gnrc_pktsnip_t *pkt)
             break;
 #endif
         default:
+            (void)hdr_len;  /* in case none of the ifdef'd cases are
```

Comment:
You shouldn't need that, because you already initialize and access `hdr_len` outside any `ifdef`?